src/cloud-cli/meltano/cloud/cli/history.py

```diff
@@ -70,27 +108,8 @@ def _format_history_table(history: list[CloudExecution], table_format: str) -> s
     headers = ["Execution ID", "Schedule Name", "Executed At", "Result", "Duration"]
 
     for execution in history:
```

Comment:
```suggestion
    table = [process_table_row(execution) for execution in for execution in history]
```

GitHub won't let the suggestion remove `table = []` above, or the 2 lines below.

Reply:
Thanks, see f765093997caaf0fb168044fcfdde2b999cf9014